libraries/stdlib/stdlib_defs.mtlx

```diff
@@ -193,8 +193,8 @@
     <input name="texcoord" type="vector2" defaultgeomprop="UV0"/>
     <input name="uvtiling" type="vector2" value="1.0, 1.0"/>
     <input name="uvoffset" type="vector2" value="0.0, 0.0"/>
-    <parameter name="realworldimagesize" type="vector2" value="1.0, 1.0" unittype="distance"/>
-    <parameter name="realworldtilesize" type="vector2" value="1.0, 1.0" unittype="distance"/>
+    <input name="realworldimagesize" type="vector2" value="1.0, 1.0" unittype="distance"/>
```

Comment:
This corrects a previous check-in. The nodegraph is has input and not parameters.